plugins/MumblePlugin.h

```diff
@@ -233,6 +233,7 @@ enum Mumble_ErrorCode {
 	MUMBLE_EC_DATA_ID_TOO_LONG,
 	MUMBLE_EC_API_REQUEST_TIMEOUT,
 	MUMBLE_EC_OPERATION_UNSUPPORTED_BY_SERVER,
+	MUMBLE_EC_LINK_CHANNEL_NOT_FOUND,
```

Comment:
Do we really have to make the differentiation whether a channel is not found in the context of channel links and everywhere else? :thinking: